oauthproxy.go

```diff
@@ -1248,3 +1251,27 @@ func (p *OAuthProxy) errorJSON(rw http.ResponseWriter, code int) {
 	// application/json
 	rw.Write([]byte("{}"))
 }
+
+// LoggingCSRFCookiesInOAuthCallback Log all CSRF cookies found in HTTP request OAuth callback,
+// which were successfully parsed
+func LoggingCSRFCookiesInOAuthCallback(req *http.Request, cookieName string) {
+	cookies := req.Cookies()
+	if len(cookies) == 0 {
+		logger.Println(req, logger.AuthFailure, "No cookies were found in OAuth callback.")
+	} else {
+		var anyFound = false
+		for i := range cookies {
+			var c = cookies[i]
```

Comment:
You're not capturing the content of the cookie so you can safely iterate like this
```suggestion
		for _, c := range cookies {
```